modules/core/src/system.cpp

```diff
@@ -1300,6 +1300,402 @@ void setUseCollection(bool flag)
 }
 #endif
 
+template<class OBJECT>
+Node<OBJECT>::Node()
+{
+    m_pParent  = 0;
+}
```

Comment:
In general, implementation of template classes should be placed in headers (.hpp or .impl.hpp)